api/track.js

```diff
@@ -2,6 +2,8 @@ import encodeUrl from 'encodeurl';
 import microSession from 'micro-cookie-session';
 import { send } from 'micro';
 import md5 from 'md5';
+const querystring = require("querystring");
```

Comment:
why require?

Reply:
'let's just hope for good enough' over 'strive for perfection' 😉 
Will change 👍